test/performance/odp_ipsec.c

```diff
@@ -584,28 +591,29 @@ static uint8_t test_data[] = {
 	0x08, 0x00, 0xfb, 0x37, 0x12, 0x34, 0x00, 0x00
 };
 
-static odp_packet_t
-make_packet(odp_pool_t pkt_pool, unsigned int payload_length)
+static int
+make_packet_multi(odp_pool_t pkt_pool, unsigned int payload_length,
+		  odp_packet_t pkt[], int num)
 {
-	odp_packet_t pkt;
+	int i, ret;
 
-	if (payload_length < sizeof(test_data))
-		return ODP_PACKET_INVALID;
-
-	pkt = odp_packet_alloc(pkt_pool, payload_length);
-	if (pkt == ODP_PACKET_INVALID) {
-		ODPH_ERR("failed to allocate buffer\n");
-		return pkt;
+	ret = odp_packet_alloc_multi(pkt_pool, payload_length, pkt, num);
+	if (ret != num) {
+		ODPH_ERR("Could not allocate buffer\n");
+		odp_packet_free_sp(pkt, ret);
```

Comment:
ret may be negative here. This should be called only if it is positive.

Reply:
Agreed. Will make this change in v2.